certify-service/src/main/java/io/mosip/certify/services/vcsigners/KeymanagerLibSigner.java

```diff
@@ -85,7 +85,7 @@ public VCResult<JsonLDObject> attachSignature(String unSignedVC) {
             throw new CertifyException("Error during canonicalization");
         }
         String vcEncodedHash = Base64.getUrlEncoder().encodeToString(vcHashBytes);
-        LdProof ldProofWithJWS = proofGenerator.generateProof(vcLdProof, vcEncodedHash);
+        LdProof ldProofWithJWS = proofGenerator.generateProof(vcLdProof, vcEncodedHash, defaultSettings);
```

Comment:
@vishwa-vyom I can move the `vcLDProof` object creation inside this `generateProof` method and make the ProofGenerator a Single Method Interface if I perform the canonicalization inside as well, but then it becomes a bit confusing to decide how to divide the concerns b/w the ProofGenerator & a VCSigner.